setup-model.sh

```diff
@@ -47,3 +60,188 @@ if [ ! -e "coco.names" ]; then
 else
     echo "You already have coco.names"
 fi
+
+
+if [ ! -e "rtmdet/end2end.engine" ]; then
+    if [ $arch != "x86_64" ] && [ $arch != "aarch64" ]; then
+        echo "Unknown architecture: $arch"
+        exit 1
+    fi
+    dc_file=$dc_dir/docker-compose-people-setup-model.yaml
+    if [ $arch = "x86_64" ]; then
+        people_service=people-setup-model
+    elif [ $arch = "aarch64" ]; then
+        people_service=people-jetson-setup-model
+    fi
```

Comment:
- people-setup-model -> people
- people-jetson-setup-model -> people-jetson

may work

Reply:
I will modify this and remove docker-compose-people-setup-model.yaml.